libnavigation-base/src/main/java/com/mapbox/navigation/base/options/PredictiveCacheZoomLevelOptions.kt

```diff
@@ -0,0 +1,85 @@
+package com.mapbox.navigation.base.options
+
+/**
+ * Maps predictive cache zoom level options.
+ *
+ * @param minZoom Minimum zoom level for the tile package. See **com.mapbox.maps.TilesetDescriptorOptions#getMinZoom**
+ * @param maxZoom Maximum zoom level for the tile package. See **com.mapbox.maps.TilesetDescriptorOptions#getMaxZoom**
+ */
+class PredictiveCacheZoomLevelOptions private constructor(
```

Comment:
Do we need a separate class? It feels like these could just be part of the map options.